### PR TITLE
added proxy services

### PR DESCRIPTION
**Pull request by ermiyas111**
Added proxy servers

### CODE REVIEW

**Comment by agebeyaw** [comment]
@ermiyas111 this PR is huge to review unless we simply approve.
If it is auto generated, can you try to reduce if there is any useless code?

**Comment by ermiyas111** [comment]
> @ermiyas111 this PR is huge to review unless we simply approve. If it is auto generated, can you try to reduce if there is any useless code?

Django has a default admin module. Most of the files are from that module. Removing them.

File: python_example_dashboard/settings.py
Comment (agebeyaw) [inline python_example_dashboard/settings.py:1]: I think some of the settings are not necessary for our case, can you clean up?

Comment (ermiyas111) [reply python_example_dashboard/settings.py:1]: cleaning up

**Pull request merged**